src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public String doWork() {
+        return "Truck is working";
```

Comment:
![image](https://user-images.githubusercontent.com/12864128/195922544-fd117a6f-0c82-44cc-933b-ba9c2f7f8aff.png)
Methods should not return anything. They must be void and print to the screen inside the method.